pkg/loop/helper_test.go

```diff
@@ -1,24 +1,39 @@
 package loop
 
+import "github.com/smartcontractkit/chainlink-relay/pkg/types"
+
 const KeepAliveTickDuration = keepAliveTickDuration
 
-func (p *RelayerService) Kill() {
-	p.mu.RLock()
-	defer p.mu.RUnlock()
-	if p.client != nil {
-		p.client.Kill()
-	}
+// TestHook returns a TestRelayerService.
+// It must only be called once, and before Start.
+func (r *pluginService[P, S]) TestHook() TestRelayerService[P, S] {
```

Comment:
what are P and S? can we colocate with other *pluginService funcs since we are in the same package?

Reply:
These are isolated in a `_test.go` file to exclude them from the real exported package. We have a habit of using `helpers_test.go` as a catch all - would `plugin_service_test.go` help?